src/compass/utils/geometry_utils.py

```diff
@@ -265,3 +265,163 @@ def get_unit_vector4component_of_interest(los_inc_angle, los_az_angle, comp='enu
         ]
 
     return unit_vec
+
+
+def enu2rgaz(radargrid_ref, orbit, ellipsoid,
+             lon_arr, lat_arr, hgt_arr,
+             e_arr, n_arr, u_arr,
+             geo2rdr_params=None):
+    '''
+    Convert ENU displacement into range / azimuth displacement,
+    based on the idea mentioned in ETAD ATBD, available in the link below:
+    ("https://sentinels.copernicus.eu/documents/247904/4629150/"
+     "ETAD-DLR-DD-0008_Algorithm-Technical-Baseline-Document_2.3.pdf/"
+     "5cb45b43-76dc-8dec-04ef-ca1252ace434?t=1680181574715")
+
```

Comment:
Can we make this one line so we can click on it, maybe adding `# noqa` at the end if you are worried about it setting up linters?